src/storage/file_io.c

```diff
@@ -11858,3 +11864,183 @@ fileio_is_formatted_page (THREAD_ENTRY * thread_p, const char *io_page)
   free_and_init (ref_page);
   return is_formatted;
 }
+
+#define	MAX_INTEGER_DISPLAY_LENGTH	  11
+#define	MAX_SHORT_DISPLAY_LENGTH	  6
+/*
+ * xmanage_lob_dir () - Create or remove a LOB directory.
+ *
+ * hfid (in) : hfid is used to identify the table.
+ * attrid_arr (in)	 : Array containing attr_id values of LOB-type columns.
+ * lob_arr_length (in)	 : Length of the attrid_arr array
+ * mode (in)	 : mode for handling the LOB directory
+ *
+ * NOTE: A LOB directory is created immediately,
+ *       whereas deletion is deferred using the log_append_postpone() function and executed at commit time.
+ */
+int
+xmanage_lob_dir (HFID * hfid, int *attrid_arr, int lob_arr_length, LOB_DIR_MANAGE_MODE mode)
+{
+#if defined(SERVER_MODE) || defined(SA_MODE)
+  char dirbuf[PATH_MAX];
+  char rv_path[PATH_MAX];
+  int ret = NO_ERROR;
+  THREAD_ENTRY *thread_p = thread_get_thread_entry_info ();
+  LOG_DATA_ADDR addr;
```

Comment:
xmanage_lob_dir()를 호출하는 상위 함수에서 THREAD_ENTRY* thread_p를 이미 얻어서 가지고 있네요. 이 함수 내에서 다시 thread_get_thread_entry_info ()로 가져올 필요 없이, 이 함수의 파라미터로 추가하면 좋을 것 같습니다.

```suggestion
int
xmanage_lob_dir (THREAD_ENTRY *thread_p, HFID * hfid, int *attrid_arr, int lob_arr_length, LOB_DIR_MANAGE_MODE mode)
{
#if defined(SERVER_MODE) || defined(SA_MODE)
  char dirbuf[PATH_MAX];
  char rv_path[PATH_MAX];
  int ret = NO_ERROR;
  LOG_DATA_ADDR addr;
```

Reply:
반영했습니다.